src/argilla/client/feedback/dataset.py

```diff
@@ -199,9 +221,10 @@ def __init__(
             raise TypeError(f"Expected `questions` to be a list, got {type(questions)} instead.")
         any_required = False
         for question in questions:
-            if not isinstance(question, (TextQuestion, RatingQuestion)):
+            if not isinstance(question, (TextQuestion, RatingQuestion, LabelQuestion, MultiLabelQuestion)):
```

Comment:
Maybe we could create a base class for the questions and just do `isinstance(question, BaseQuestion)`

Reply:
I've seen that we could use `QuestionSchema`, but if for some reason a user creates a class inheriting from it, it would pass this validation and we would have some errors later.

Reply:
We wanted to specifically constraint this to the supported questions in the Python client, that's why :(